src/responses.rs

```diff
@@ -314,19 +314,14 @@ pub struct FailureItem {
 /// 'pay' command
 #[derive(Debug, Clone, Deserialize, Serialize)]
 pub struct Pay {
-    pub id: u64,
     pub payment_hash: String,
     pub destination: String,
     pub msatoshi: u64,
     pub msatoshi_sent: u64,
-    pub created_at: u64,
+    pub created_at: f64,
     pub status: String,
     pub payment_preimage: String,
-    pub description: String,
-    pub getroute_tries: u64,
-    pub sendpay_tries: u64,
-    pub route: Vec<common::RouteItem>,
-    pub failures: Vec<FailureItem>,
+    pub parts: u64,
```

Comment:
It is called `number_of_parts` in the c-lightning code

https://github.com/ElementsProject/lightning/blob/a027b3a80cad8cd72c8629d44d687c48d9bfb181/plugins/pay.c#L1783